src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -406,6 +407,7 @@ public VoldemortConfig(Props props) {
 
         this.modifiedProtocol = props.getString("readonly.modify.protocol", null);
         this.modifiedPort = props.getInt("readonly.modify.port", -1);
+        this.bouncyCastleEnabled = props.getBoolean("readonly.bouncy.castle.enabled", false);
```

Comment:
VoldemortServer constructor seems like a good place. 

There are two constructors, one for test and other for actual flow, so you need to put it at both the places.